content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -242,24 +244,20 @@ status:
 ```
 
 The new Pod conditions must comply with Kubernetes [label key format](/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).
-Since the `kubectl patch` command still doesn't support patching object status,
-the new Pod conditions have to be injected through the `PATCH` action using
-one of the [KubeClient libraries](/docs/reference/using-api/client-libraries/).
+Since the `kubectl patch` command doesn't support patching object status,
+applications (or [Operators](/docs/concepts/extend-kubernetes/operator/)) that want to
+set custom Pod conditions must use the `PATCH` action. You can use a
+[Kubernetes client library](/docs/reference/using-api/client-libraries/) to write code
+that sets custom Pod conditions for Pod ready++.
 
-With the introduction of new Pod conditions, a Pod is evaluated to be ready **only**
+For a Pod that's using custom conditions, that Pod is evaluated to be ready **only**
 when both the following statements are true:
 
 * All containers in the Pod are ready.
 * All conditions specified in `ReadinessGates` are "`True`".
 
-To facilitate this change to Pod readiness evaluation, a new Pod condition
-`ContainersReady` is introduced to capture the old Pod `Ready` condition.
-
-In K8s 1.11, as an alpha feature, the "Pod Ready++" feature has to be explicitly enabled by
-setting the `PodReadinessGates` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-to true.
-
-In K8s 1.12, the feature is enabled by default.
+When a Pod's containers are Ready but at least one custom condition is missing or
```

Comment:
nit: Does this help clarify?
When a Pod's containers are in the `Ready` state [or condition] ...

Reply:
The pod's `containerdsready` condition will be set to true when:

Reply:
Suggest changing this to "The pod's `ContainersReady condition will be set to true when a Pod's containers are in the `Ready` state [or condition] but at least one custom condition is missing or `False`"